views/index.jade

```diff
@@ -23,19 +23,21 @@ block content
     div.container
       div.row
         div.col-md-6.whatis
-          h2 Build unstoppable applications
+          h2 Welcome to the Ethereum open source community
```

Comment:
I'm with you.  It is bland.   Here's why I think it's still necessary:
Whether you look at Linux.org ("Welcome to Our Community") or signs throughout the world  ("Seja bem vindo!"), the greeting invites you and says "You are important to us!"
 
Technical people in general and crypto people in specific tend to be brash and exclusionary.  Let's be different.

Yes, we would like to get in zingers as to why we're the smartest and best, but visitors don't care how much we know until they know how much we care.